src/autotype/CMakeLists.txt

```diff
@@ -13,5 +13,8 @@ if(UNIX AND NOT APPLE)
 endif()
 
 if(WITH_TESTS)
-  add_subdirectory(test)
-endif()
+  # autotype non supported on Windows right now
+  if(UNIX)
```

Comment:
My apologies, I should have realized WHERE this CMAKE file was located :stuck_out_tongue_winking_eye: 

This implementation is good to go. Although I do think we should migrate all tests to the test folder and differentiate by platform those tests that are platform specific.